lib/mongo/collection/view/change_stream.rb

```diff
@@ -176,15 +249,31 @@ def for_collection?
         end
 
         def cache_resume_token(doc)
+          # Always record both resume token and operation time,
+          # in case we get an older or newer server during rolling
+          # upgrades/downgrades
           unless @resume_token = (doc[:_id] && doc[:_id].dup)
-            raise Error::MissingResumeToken.new
+            raise Error::MissingResumeToken
           end
+          @start_at_operation_time = doc['clusterTime']
```

Comment:
This should be doc['opreationTime'] I think

Reply:
There is no operationTime in this doc.